### DISCUSSION TITLE
.net for Android & .net for iOS

### DISCUSSION THREAD

**Discussion by MateuszLas421** [reactiveui/ReactiveUI#3810] [Q&A]
Is there planned support for .NET for iOS/Android projects (.NET 8.0) after Xamarin is deprecated?

**Comment by ChrisPulman** [comment]
The Core ReactiveUI package has Net 8 Android and iOS targets, if you feel something is missing please let us know, thank you. Previously additional packages were made to add the Xamarin base and relevant functionality

**Reply by ChrisPulman** [reply]
There's also Maui as the replacement for Xamarin which supports both.

**Reply by MateuszLas421** [reply]
In my opinion, there's a lack of migration form ReactiveUI.AndroidX to ReactiveUI, especially since Google has been saying for a long time that the 'android.support.*' libraries are outdated.

 https://developer.android.com/topic/libraries/support-library

And those 'android.support.*' libraries haven't been updated for a long time.
And in the latest version, we have ReactiveFragment, which is based on Fragment. 
https://developer.android.com/reference/android/app/Fragment
`This class was deprecated in API level 28.
Use the [Jetpack Fragment Library](https://developer.android.com/jetpack) [Fragment](https://developer.android.com/reference/androidx/fragment/app/Fragment.html) for consistent behavior across all devices and access to [Lifecycle](https://developer.android.com/topic/libraries/architecture/lifecycle).`
